metron-interface/metron-alerts/src/app/shared/modal-loading-indicator/modal-loading-indicator.component.scss

```diff
@@ -15,7 +15,26 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-"use strict";
-exports.environment = {
-    production: true
-};
```

Comment:
Git get confused on this.
environment.prod.js was deleted as an unused file
scss was introduced for the modal loader.